engine/src/main/java/org/hibernate/validator/internal/metadata/descriptor/ConstraintDescriptorImpl.java

```diff
@@ -75,7 +76,8 @@
 			Retention.class.getName(),
 			Target.class.getName(),
 			Constraint.class.getName(),
-			ReportAsSingleViolation.class.getName()
+			ReportAsSingleViolation.class.getName(),
```

Comment:
Cool. Wondering a bit why we use the class names instead of the classes themselves here. But that's for another time.